e2e/puppeteer-custom-environment.ts

```diff
@@ -1,13 +1,23 @@
 const PuppeteerEnvironment = require('jest-environment-puppeteer');
 const fs = require('fs-extra');
+const path = require('path');
 
 require('jest-circus');
 
+// TODO Replace PuppeteerEnvironment with NodeEnvironment
```

Comment:
nit: per new style guide, please include a ticket reference in the TODO if you have one: https://github.com/all-of-us/workbench/wiki/UI-Style-Guide#code-comments

Reply:
👍🏻